docs/modules/Social Login/particleAuth.md

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_position: 2
+---
+
+# Biconomy and Particle Auth
+
+Particle Auth provides MPC (Multi-Party Computation)-based threshold signatures. This integration provides the auth module with the smart-account package combined.
+
+## Code Examples
+
+- [Implement Particle Auth With React](https://github.com/bcnmy/sdk-examples/tree/master/react-vite-particle-auth)
```

Comment:
I think we should include Biconomy as well

[Implement Particle Auth with Biconomy SDK using React](Link)